static/test/javascripts/spec/common/onward/breaking-news.spec.js

```diff
@@ -52,13 +52,16 @@ define([
                         webTitle: 'Breaking News',
                         collections: collections
                     })]);
-                    breakingNews().then(function(result) {
+                    Promise.resolve().then(function () {
```

Comment:
based on @piuccio's recommendation, that

> The first Promise.resolve() is extra safety in case breakingNews() throws an exception. 

seems sensible to me?